plugins/workbench-resources/src/components/Navigator.svelte

```diff
@@ -105,6 +110,10 @@
 
     {#if topSpecials.length > 0 || bottomSpecials.length > 0}<TreeSeparator />{/if}
 
+    {#if starred.length}
+      <StarredNav label={preference.string.Starred} spaces={starred} on:space {currentSpace} />
```

Comment:
Could we do same as for Archived, make this optional and contributable? in separate PR.